### ISSUE TITLE
Block-Level widget disappears on drag/drop to some other block-level widget, if user tries to drag one block level widget and drop into another widget

### ISSUE THREAD

**Issue by usmannoor10p** [ckeditor/ckeditor4#4380]
I am attaching the zip file for the code. Please see this and do tell me what's wrong with it.

**Actual Result**:
Block-level widgets merging into one another on drag and drop.

**Expected Result**:
The block-level widget should not get merged into any other block-level widget on the **drop**.

Your help will be appreciated a lot.

![ezgif com-gif-maker](https://user-images.githubusercontent.com/74644877/99498615-96b34c00-2999-11eb-9528-e760d4ceee51.gif)

[Ckeditor.zip](https://github.com/ckeditor/ckeditor4/files/5558332/Ckeditor.zip)

**Comment by f1ames** [comment]
Hi @usmannoor10p,

You can see that such behavior (widgets merging on drag and drop) is not happening with official CKEditor 4 widgets (e.g. [Enhanced Image](https://ckeditor.com/docs/ckeditor4/latest/examples/image2.html) or [Simple Box sample widget](https://ckeditor.com/docs/ckeditor4/latest/examples/simplebox.html)).

From what I see you are using custom widgets - if they are provided by a 3rd-party please report an issue to the plugin authors. If you are the author, please leave your question on [Stack Overflow](https://stackoverflow.com/questions/tagged/ckeditor) where our team and the community assist other users in solving their issues. Since CKEditor is an Open Source software, we believe it’s crucial to share proposed solutions and make others benefit from them.

If you have CKEditor license, please contact our support team.

**Comment by usmannoor10p** [comment]
Hi @f1ames ,

Thanks for your response. even if you remove all the plugins and then try, you will get the same.
As far as I have tested the main cause is the `allowedContent=true`, by removing it the drag-drop works properly on block-level widgets but all the CSS got removed. I am attaching the screenshot below.

![image](https://user-images.githubusercontent.com/74644877/99527870-4949d580-29bf-11eb-92b5-960e73845a4c.png)
